Makefile

```diff
@@ -2,7 +2,7 @@
 
 IMAGE := uk.ac.wellcome/web-scraper
 ECR_IMAGE := 160358319781.dkr.ecr.eu-west-1.amazonaws.com/$(IMAGE)
-VERSION := 2018.10.0
+VERSION := 2018.12.0
```

Comment:
what happened to 11?

Reply:
Good question - The convention we're following here is {YYYY}.{MM}.{SERIAL_NO} - we made releases in October and December but not November.

Reply:
👍